src/Components/test/testassets/ComponentsApp.App/ComponentsApp.App.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
+  <!-- Reference targets and props from Components.Build package -->
+  <Import Project="..\..\..\Build\src\ReferenceFromSource.props" />
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
```

Comment:
This was using the wrong build targets. Regardless, writing an explicit Import is way easier to follow. Took me a while to realize setting this property resulted in side-effects.